### PR TITLE
extended homepage

### PR DESCRIPTION
**Pull request by tusharkandpal**
- moved iframe styles to global.css
- quick introduction
- Wings UI setup link
- font-awesome link

### CODE REVIEW

File: utilities/global.css [hunk 1]
```diff
@@ -25,6 +25,19 @@ a {
   text-decoration: none;
   color: black;
 }
+.iframe-container {
+  position: relative;
+  overflow: auto;
+  min-height: 50vh;
+}
+iframe {
+  border: 0;
```
Comment (Priya-Patel31) [inline utilities/global.css]: use border:none instead of border: 0

File: utilities/global.css [hunk 2]
```diff
@@ -25,6 +25,19 @@ a {
   text-decoration: none;
   color: black;
```
Comment (Priya-Patel31) [inline utilities/global.css]: use color variable for these colors.

**Review by Priya-Patel31 — APPROVED** [review]
Looks good and you can merge it .

File: utilities/global.css [hunk 3]
```diff
@@ -12,18 +12,33 @@
...
+  position: relative;
+  overflow: auto;
+  min-height: 50vh;
+}
+iframe {
+  border: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 1000px;
```
Comment (ShreyashMahajan) [inline utilities/global.css]: use rem for more than 10px

**Review by ShreyashMahajan — APPROVED** [review]
UI looks clean, Good to merge

File: utilities/global.css (hunk 3 above)
Comment (tusharkandpal) [reply utilities/global.css]: ya done thanks!

**Pull request merged**